src/js/types/beacon.ts

```diff
@@ -162,6 +162,39 @@ export interface BeaconServiceInfo {
   info?: GenericInfoField;
 }
 
+// ----------------------------
+// filters
+// ----------------------------
+
+// https://docs.genomebeacons.org/filters/
+export interface BeaconFilteringTermsResponse {
+  meta?: {
+    beaconId: string;
+  };
+  response?: {
+    filteringTerms: BeaconFilteringTermFromResponse[];
+  };
+}
+
+export interface BeaconFilteringTermFromResponse {
+  type: 'alphanumeric';
+  id: string; // something more here?
+  label: string;
+  description: string;
+  values: string[];
+  bento: {
+    section: string;
+  };
+  units?: string;
+}
+
+export type BeaconFilterForQuery = Omit<BeaconFilteringTermFromResponse, 'bento'>;
+
+export interface BeaconFilterSection {
+  section_title: BeaconFilteringTermFromResponse['bento']['section'];
```

Comment:
- camel case 
- BeaconFilteringTermFromResponse['bento']['section'] just evaluates to a string so should be just labelled as string

Reply:
if you really want to provide a relation you can add a comment

Reply:
Fixed, although note that the snake case is borrowed from search.ts.

Reply:
ah, in fact the dependency doesn't really need to be there anymore, I can rewrite a few other types to get rid of this.

Reply:
Can we change them all at once in a separate pr? 

If we change beacon section_title and regular search section_title at different times there's going to be a bunch of weird intermediate types needed (since the beacon network will need a type that's essentially `SectionButWithCamelCase`... which will be deprecated almost immediately when the fix is applied everywhere).